src/components/Comment/Comment.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Comment.scss';
+import { CommentShape } from '../shapes/CommentShape';
+
+export const Comment = ({ name, body, email }) => (
+  <div className="comment">
+    <div className="comment__user">
+      {name}
+    </div>
+    <div className="comment__user">
+      <a href="mailto:email">
```

Comment:
напевне єдина помилка але не критична для цього завдання [скрін](https://i.imgur.com/eBCpzWr.png)
```suggestion
      <a href={'mailto:' + email}>
```